examples/flutter_gallery/lib/demo/video_demo.dart

```diff
@@ -0,0 +1,361 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'package:connectivity/connectivity.dart';
+import 'package:flutter/material.dart';
+import 'package:video_player/video_player.dart';
+
+// TODO(sigurdm): These should not be stored here.
+const String butterflyUri =
+    'https://flutter.github.io/assets-for-api-docs/videos/butterfly.mp4';
+
+const String beeUri =
+    'https://flutter.github.io/assets-for-api-docs/videos/bee.mp4';
+
+class VideoCard extends StatelessWidget {
+  final VideoPlayerController controller;
+  final String title;
+  final String subtitle;
+
+  const VideoCard({Key key, this.controller, this.title, this.subtitle})
+      : super(key: key);
```

Comment:
Could be
```dart
const VideoCard({
  Key key,
  this.controller,
  this.title,
  this.subtitle,
}) : super(key: key);
```
though I don't know if `dartfmt` leaves it that way.

Reply:
Dartfmt ruins this. Leaving as is.

Reply:
I filed https://github.com/dart-lang/dart_style/issues/658